.gitignore

```diff
@@ -15,7 +15,6 @@ cleanup.sh
 .env*
 .temp/
 
-.vscode
 *.err
```

Comment:
any specific reason on why removing `.vscode` from here?

Reply:
Good catch. This was a mistake. Added it back